css-brochure-form/styles.css

```diff
@@ -0,0 +1,70 @@
+.form-container {
+  margin: auto;
+  width: 50%;
+}
+
+h2 {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  text-align: center;
+  color: #f25c54;
+  text-transform: uppercase;
+  font-size: 24px;
+}
+
+fieldset {
+  border: none;
+}
+
+label {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 2em;
+  text-align: left;
+  font-size: 16px;
+  padding-left: 10px;
+  letter-spacing: 1px;
+}
+
+div > input {
+  border-radius: 0.5em;
+  width: 600px;
+  height: 50px;
+  border: 2px solid #e1e5ea;
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 18px;
+  padding-left: 10px;
+}
+
+select {
+  border-radius: 0.3em;
+  width: 600px;
+  height: 50px;
+  border: 2px solid #e1e5ea;
+  background-color: #f5f3f3;
+  color: black;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  padding-left: 5px;
+}
+
+button {
+  background-color: #f25c54;
+  border-radius: 2.75em;
+  color: white;
+  border: none;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  width: 130px;
+  height: 50px;
+  margin-left: 470px;
```

Comment:
Woah, that's a lot of margin!! If you are doing this to place the button on the right side of the form, you should just apply `text-align: right` to the `div` that the `button` is a child of.

Reply:
![Screen Shot 2021-01-22 at 5 29 23 PM](https://user-images.githubusercontent.com/74885918/105564528-72485880-5cd7-11eb-9c41-86595165a433.png)